docs/source/spatialstats.rst

```diff
@@ -0,0 +1,143 @@
+- propagate errors between spatial ensembles at different scales (e.g., sum of glacier volume changes).
+
+.. contents:: Contents 
+   :local:
+
+Assumptions for statistical inference in spatial statistics
+***********************************************************
+
+Spatial statistics are valid if the variable of interest verifies `the assumption of second-order stationarity
+<https://www.aspexit.com/en/fundamental-assumptions-of-the-variogram-second-order-stationarity-intrinsic-stationarity-what-is-this-all-about/>`_.
+That is, if the three following assumptions are verified:
+
+1. The mean of the variable of interest is stationary in space, i.e. constant over sufficiently large areas,
+2. The variance of the variable of interest is stationary in space, i.e. constant over sufficiently large areas.
+3. The covariance between two observations only depends on the spatial distance between them, i.e. no other factor than this distance plays a role in the spatial correlation of measurement errors.
+
+A sufficiently large averaging area is an area expected to fit within the spatial domain studied.
+
+In other words, for a reliable analysis, the DEM should:
+
+1. Not contain systematic biases that do not average out over sufficiently large distances (e.g., shifts, tilts), but can contain pseudo-periodic biases (e.g., along-track undulations),
+2. Not contain measurement errors that vary significantly in space.
+3. Not contain factors that significantly affect the distribution of measurement errors, except for the spatial distance.
+
+Quantifying the precision of a single DEM, or of a difference of DEMs
+*********************************************************************
+
+To statistically infer the precision of a DEM, the DEM has to be compared against independent elevation observations.
+
+If the other elevation data is known to be of higher-precision, one can assume that the analysis of differences will represent the precision of the rougher DEM.
+Otherwise, significant measurement errors can originate from both sets of elevation observations, and the analysis of differences will represent the mixed precision of the two.
+
+TODO: complete with Hugonnet et al. (in prep)
+
+Stable terrain: proxy for infering DEM precision
+************************************************
+
+When comparing elevation datasets, stable terrain is usually used a proxy
+
+Metrics for DEM precision
+*************************
+
+The precision of DEMs has generally been reported as a single value indicating the random error at the scale of a single pixel, for example :math:`\pm 2` meters.
+
+However, the significant variability of elevation measurement errors
+
+Pixel-wise elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+
+Spatially-integrated elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The standard error (SE) of a statistic is the standard deviation of the distribution of this statistic.
```

Comment:
This is also described here: https://xdem.readthedocs.io/en/latest/spatial_stats.html#standard-error